savant/app/db_config.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
This is good. However you need to call it `db_config_travis.json`. Use your Travis configuration to copy it to `db_config.json` when the test starts.

Reply:
Does this mean that db_config.json is empty?